packages/twenty-server/src/database/typeorm/metadata/migrations/1725893697807-addIndexTypeColumnToIndexMetadata.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddIndexTypeColumnToIndexMetadata1725893697807
+  implements MigrationInterface
+{
+  name = 'AddIndexTypeColumnToIndexMetadata1725893697807';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TYPE metadata.INDEX_TYPES AS ENUM ('BTREE', 'GIN')`,
```

Comment:
Just realised but I think that's not the naming convention we follow currently.
We put the name of the table as well. Can you double check please? 🙏